frontend/src/components/ShootWorkers/ContainerRuntime.vue

```diff
@@ -70,50 +64,63 @@ export default {
     machineImageCri: {
       type: Array,
       default: () => []
+    },
+    kubernetesVersion: {
+      type: String,
+      required: true
     }
   },
   data () {
     return {
       validationErrors,
-      valid: undefined,
-      containerRuntime: undefined,
-      ociRuntime: undefined
+      valid: undefined
     }
   },
   validations,
   computed: {
     containerRuntimeItems () {
-      const containerRuntimes = map(this.machineImageCri, 'name')
-      return uniq([...containerRuntimes, DEFAULT_CONTAINER_RUNTIME])
+      return map(this.machineImageCri, 'name')
```

Comment:
I would propose to rename this property to `criNames`

Reply:
ok, done